code/game/gamemodes/handofgod/_handofgod.dm

```diff
@@ -316,6 +316,18 @@ var/global/list/global_handofgod_structuretypes = list()
 			return 1
 
 
+/mob/camera/god/proc/is_handofgod_myfollowers(A)
+	if(!ishuman(A))
+		return 0
```

Comment:
why do you need to check whether it's a human?
Also, since this proc is currently only used in one place, we already know the first argument is a mob/living/carbon.